### PR TITLE
Update setValidSelect to be an arrow function

### LINKED ISSUE #3509: Update setValidSelect to be an arrow function
Update setValidSelect.js in the html/utilities/validation directory to be an arrow function.
PRs should be made against the staging branch.

### PR DESCRIPTION
**Pull request by mantasio**
## What does this PR do?
Updated setValidSelect to be an arrow function

### Associated Issue
Fixes #3509

### CODE REVIEW

File: html/utilities/validation/setValidSelect.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function setValidSelect(inputContainer) {
```
Comment (guern1kn) [inline html/utilities/validation/setValidSelect.js:1]: @mantasio Thanks for the PR! Let's get a semi-colon added to the end of this function and we'll get it merged!

**Review by guern1kn — CHANGES_REQUESTED** [review]

File: html/utilities/validation/setValidSelect.js (hunk 1 above)
Comment (mantasio) [reply html/utilities/validation/setValidSelect.js:1]: sure, done!

**Review by afebbraro — APPROVED** [review]

**Review by guern1kn — APPROVED** [review]
Thanks for contributing @mantasio !

File: html/utilities/validation/setValidSelect.js (hunk 1 above)
Comment (guern1kn) [reply html/utilities/validation/setValidSelect.js:1]: @mantasio Last thing, could you update this branch with staging?

Comment (mantasio) [reply html/utilities/validation/setValidSelect.js:1]: @guern1kn done!

**Pull request merged**